packages/server/src/modules/reports/typeDefs/yearly-ledger.graphql.ts

```
@@ -0,0 +1,37 @@
+    yearlyLedgerReport(year: Int!): YearlyLedgerReport! @auth(role: ACCOUNTANT)
+  }
+
+  " yearly ledger report "
+  type YearlyLedgerReport {
+    id: ID!
+    year: Int!
+    financialEntitiesInfo: [YearlyLedgerReportFinancialEntityInfo!]!
+  }
+
+  " Vat report record "
+  type YearlyLedgerReportFinancialEntityInfo {
```

ESLint

[error] type "YearlyLedgerReportFinancialEntityInfo" must have exactly one non-nullable unique identifier.
Accepted name: id.
Accepted types: ID or UUID.

(@graphql-eslint/strict-id-in-types)